### PR TITLE
Fix #1287: Update our-team.html

### LINKED ISSUE #1287: Fix team page content placement and modify hyperlinks
- Observed behavior

1) The heading 'Core Team' is hiding behind the navigation-bar.

laptop

phone

2) All the hyperlinks in team-cards open up in the same tab which is hindering the user-experience.

- Expected behavior 

1) The content of team page should be placed a little lower. (Adding margin-top to the content will work)

2) Hyperlinks should open up in another tab. (Adding target attribute to the anchor tag will solve this)

### PR DESCRIPTION
**Pull request by geekayush**
Fixes #1287  @RishabhJain2018

### CODE REVIEW

**Comment by RishabhJain2018** [comment]
@geekayush Can you please paste the screenshot ?

**Comment by geekayush** [comment]
@RishabhJain2018 Here are the particular screenshots

laptop2
phone2

File: frontend/src/css/modules/teams.scss [hunk 1]
```diff
@@ -150,11 +150,11 @@ span.link-team-user {
 }
 
 .team-heading {
-    margin-top: -5px;
-    margin-bottom: 30px;
+    margin: 30px;
```
Comment (RishabhJain2018) [inline frontend/src/css/modules/teams.scss]: I think simply removing `margin-top` will solve the issue. can you please try that.

**Comment by RishabhJain2018** [comment]
@geekayush Any update on this ?

File: frontend/src/css/modules/teams.scss (hunk 1 above)
Comment (geekayush) [reply frontend/src/css/modules/teams.scss]: I did the same at first but it was not fully responsive. Example: For a screen resolution 768x642, it gave the following result.

untitled

Comment (RishabhJain2018) [reply frontend/src/css/modules/teams.scss]: Ahh..I see.

Comment (aka-jain) [reply frontend/src/css/modules/teams.scss]: @geekayush please use media query to achieve this responsive result. cc @RishabhJain2018

**Comment by aka-jain** [comment]
@RishabhJain2018 Is this ready to merge?

**Comment by RishabhJain2018** [comment]
@aka-jain No, it isn't.
@geekayush Can you add the media queries ?

**Comment by geekayush** [comment]
@RishabhJain2018  Okay, on it.

**Comment by deshraj** [comment]
@RishabhJain2018 is this ready to merge?

**Comment by RishabhJain2018** [comment]
@deshraj No, it is not.
@geekayush Can you please handle the case for all the screen sizes.

**Comment by aayusharora** [comment]
Hi @geekayush what you need to add is a media query to center the images on the screen for mobiles:
Use ` .img-class {
         display: block;
         margin: 0 auto;

}`

**Comment by RishabhJain2018** [comment]
@geekyayush Any update on this ?

**Comment by geekayush** [comment]
@RishabhJain2018 I am not able to get what you are asking. Can you please explain?

**Comment by aka-jain** [comment]
Did some changes, @RishabhJain2018 check once please.

**Pull request merged**